pages/common/git-diff.md

```diff
@@ -2,21 +2,25 @@
 
 > Show changes to tracked files.
 
-- Show changes to tracked files:
+- Show unstaged, uncommitted changes:
 
-`git diff {{PATHSPEC}}`
+`git diff`
 
-- Show only names of changed files:
+- Show all uncommitted changes (including staged ones):
 
-`git diff --name-only {{PATHSPEC}}`
+`git diff HEAD`
 
-- Output a condensed summary of extended header information:
+- Show only staged (added, but not yet committed) changes:
 
-`git diff --summary {{PATHSPEC}}`
+`git diff --staged`
 
-- Show staged (added, but not yet committed) changes only:
+- Show only names of changed files since a given commit:
 
-`git diff --staged`
+`git diff --name-only {{commit}}`
+
+- Output a summary of file creations, renames and mode changes since a given commit:
+
+`git diff --summary {{commit}}`
 
 - Create a patch file:
 
```

Comment:
Should we put the ".patch" out of the token ? IIRC, reviewboard/gerrit accepts only ".patch" files. Or I might be wrong.